client/core/types.go

```diff
@@ -640,7 +650,7 @@ func (a *dexAccount) setupCryptoV2(creds *db.PrimaryCredentials, crypter encrypt
 	}
 
 	// Prepare the chain of child indices.
-	kids := make([]uint32, 0, 10) // 1 x purpose, 1 x version (incl. oddness), 8 x 4-byte uint32s.
+	kids := make([]uint32, 0, 10) // 1 x purpose, 1 x version (incl. oddness), 7 x 4-byte uint32s, 1 x acct key index.
```

Comment:
Isn't it still 8 r-byte uint32s?

Reply:
Hmm. It is. Wonder how I arrived at 7 before! Will undo.